test/integration/framework/master_utils.go

```diff
@@ -204,6 +204,9 @@ func startMasterOrDie(masterConfig *controlplane.Config, incomingServer *httptes
 		)
 	}
 
+	if masterConfig.ExtraConfig.ServiceIPRange.IP == nil {
+		masterConfig.ExtraConfig.ServiceIPRange = net.IPNet{IP: net.ParseIP("10.0.0.0"), Mask: net.CIDRMask(24, 32)}
```

Comment:
nit: Can we import this value from https://github.com/kubernetes/kubernetes/blob/3a79bb68bd15903effbdb486619637e9598df5e3/pkg/kubeapiserver/options/options.go#L29 instead?

Reply:
Since only ```ServiceIPRange()``` is currently using ```DefaultServiceIPCIDR```, isn't ```DefaultServiceIPCIDR``` deleted when default service cluster IP is deprecated?

Reply:
Ahh sorry missed that earlier. LGTM